server/data-base/inventory.json

```diff
@@ -5,22 +5,50 @@
   "novelties": [
     {
       "category": "Shirts and Sweaters",
-      "items": []
+      "items": [
+        {
+          "item": "Cheat Sheet Tee",
+          "price": "$20.00",
+          "description": "Available in S/M/L/XL",
+          "image": "/assets/img/inventory/novelties/shirts/cheatsheet.jpg"
```

Comment:
i think we're running into the same issue of accidentally commiting images as last time - would we be able to remove the images from this PR (by moving them into the `uploads` folder)?

Reply:
Oh I purposely added them this time for testing purposes. Should I still remove?

Reply:
oh as long as you're aware, feel free to keep for now!